addons/website_event_booth/security/event_booth_security.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo><data noupdate="1">
+
+    <record id="ir_rule_event_booth_public" model="ir.rule">
```

Comment:
Shouldn't we also have the ir.model.access to give read access to public/portal, as default is 0,0,0,0 ?